assets/css/css/reset.css

```diff
@@ -0,0 +1,134 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
@edumoran This file is not used, can we remove it?

Reply:
checked